crypto/ec/ec_mult.c

```diff
@@ -236,6 +238,21 @@ int ec_scalar_mul_ladder(const EC_GROUP *group, EC_POINT *r,
         }
     }
 
+    /*
+     * Create random values for BN_consttime_swap_randomized, and for
+	 * constant-time swapping of Z_is_one in EC_POINT_SWAP below
+	 * This randomizatin is needed to prevent side channel attacks
+     * (for details, see: "Nonce@Once: A Single-Trace EM Side Channel Attack on
+     *  Several Constant-Time Elliptic Curve Implementations in Mobile Platforms"
+     * by M. Alam, B. Yilmaz, F. Werner, N. Samwel, A. Zajic, D. Genkin, Y. Yarom,
+     * and M. Prvulovic, in IEEE Euro S&P 2021)
+     */
+    BN_ULONG *swap_rand=OPENSSL_malloc((cardinality_bits+2)*sizeof(BN_ULONG));
```

Comment:
Don't mix declarations and code. Please declare `swap_rand` at the beginning of the function

Reply:
Code style: spaces around operators( =, +,*), i.e.

````
    BN_ULONG *swap_rand = OPENSSL_malloc((cardinality_bits + 2) * sizeof(BN_ULONG));
````

Reply:
Probably we need to check for a NULL return from OPENSSL_malloc

Reply:
fixing style, moving the decl up, and adding NULL check in the new PR.